netdata/netdata
# Update libbpf version

##### Summary
This PR is bringing the latest `libbpf` to Netdata.

##### Test Plan

1. Compile this branch and start netdata
2. Run the following command to verify that you do not have unexpected errors:
```sh
$ grep libbpf /var/log/netdata/*
``` 
3. Verify that charts were loaded:

```sh
$ curl -k  "$PROTOCOL"://$HOST:19999/api/v1/charts?all |  grep -w "\"plugin\": \"ebpf.plugin" | wc -l
```

##### Additional Information
| Linux Distribution | kernel version | 
|--------------------|----------------|
| Slackware Current  |     5.17.6     | 
| Arch Linux         |  5.17.1-arch4 | 
| Slackware 15.0     |     5.15.27    |
| Manjaro 21.1       |    5.10.109-1  |
| Ubuntu 20.04       |    5.4.0-97   | 
| Rocky 8.5          | 4.18.0-372.9.1.el8.x86_64 | 

<details> <summary>For users: How does this change affect me?</summary>
Describe the PR affects users: 
- Which area of Netdata is affected by the change? ebpf.plugin
- Can they see the change or is it an under the hood? If they can see it, where? It is not visible, because we are only bumping version
- How is the user impacted by the change? The new release brings improvements and new helpers that we can use.
</details>
